x/observer/client/cli/tx_add_observer.go

```diff
@@ -10,7 +13,7 @@ import (
 
 func CmdAddObserver() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "add-observer [observer-address] [zetaclient-grantee-pubkey]",
+		Use:   "add-observer [observer-address] [zetaclient-grantee-pubkey] [add_node_account_only]",
```

Comment:
I agree, but this flag is temporary only, the only reason we are doing this in two steps as because we don't have the post-generation tasks ready, and we need to push the upgrade out soon. This flag will probably be removed in the next release.